modules/backend/widgets/form/assets/js/october.form.js

```diff
@@ -164,6 +165,28 @@
         })
     }
 
+    /*
+     * Set the `active` class to the tabs for fragment urls.
+     */
+    FormWidget.prototype.activeTabs = function() {
+        // Check all tabs on page
+        $('.control-tabs .nav-tabs li a').each(function () {
+            // If the `fragment` url matches a tab `href` on the page then overwrite the `active` class for that tab.
+            if (window.location.hash === $(this).attr('href')) {
+                // Remove all `active` classes
+                $('[href*="' + window.location.hash + '"]').closest('.nav-tabs').find('li').removeClass('active');
+                $('[href*="' + window.location.hash + '"]').closest('.control-tabs').find('.tab-content div.tab-pane').removeClass('active');
+
+                // Add `active` class to matched tab
+                $('[href*="' + window.location.hash + '"]').parent('li').addClass('active');
+                $($(this).attr('data-target')).addClass('active');
```

Comment:
Does this still support nested tabs? I.e. if you have a page with a nestedform displayed in a tab, and then that nestform itself has tabs, does that still work?

<img width="1440" alt="staff members profile" src="https://user-images.githubusercontent.com/7253840/102538238-aba2fe00-4071-11eb-91ca-eb6609e11102.png">